tests/test_versions.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from meshkernel import MeshKernel
+from meshkernel.version import __backend_version__, __version__
+
+
+def test_get_meshkernel_version():
+    """Tests if we can get the version of MeshKernel through the API"""
+    mk = MeshKernel()
+    meshkernel_version = mk.get_meshkernel_version()
+    last_dot = meshkernel_version.rfind(".")
+    assert meshkernel_version[:last_dot] == __backend_version__
```

Comment:
This will not work on unmerged release branches!

When it is release time, the first thing we do is modify `__backend_version__ `. 

Release branches are tagged and merged to master after __backend_version__ is set. `mk.get_meshkernel_version()` will get the version from the backend, which always behind am unmerged release branch. 

Will revert.